src/Sarif.Viewer.VisualStudio/App.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+  </startup>
+  <appSettings>
+    <add key="TelemetryInstrumentationKey_Debug" value="80a368bb-f285-46eb-bb20-3e424593ec56"/>
```

Comment:
>80a368bb-f285-46eb-bb20-3e424593ec56 [](start = 56, length = 36)

Do we want the same GUID for debug vs. release? The nice thing about two distinct guids is that we can debug/test against one and not compromise release telemetry.

Reply:
Yes that is the one outstanding to-do. I can't create a new instance so I'll need you to do that for me.

Reply:
ok, open an issue on it for chris faucon

---
In reply to: [158073364](https://github.com/Microsoft/sarif-sdk/pull/729#discussion_r158073364) [](ancestors = 158073364)